src/humanReadablePane.js

```diff
@@ -86,6 +81,28 @@ module.exports = {
       console.log('humanReadablePane: unknown content-type?')
     }
 
+    //  @@ When we can, use CSP to turn off scripts within the iframe
+    div.setAttribute('class', 'docView')
+    const iframe = myDocument.createElement('IFRAME')
+    let dataUri
+
+    // render markdown to html
+    const markdownHtml = function () {
+      kb.fetcher.webOperation('GET', subject.uri).then(response => {
```

Comment:
**question** why do we need another fetch here? The resource should already have been fetched, hasn't it?

Reply:
The original fetch is a HEAD only to find the contentType. We need the content now to feed the `data:uri`.